views/campgrounds/show.ejs

```diff
@@ -1,8 +1,8 @@
 <%layout('layouts/boilerplate')%>
     <link rel="stylesheet" href="/stylesheets/stars.css">
 
-    <div class="row" id="showinfo">
-        <div class="col-6">
```

Comment:
Replace col-6 with col-md-6 in line number 5 and 56.
This will fix the issue.

Reply:
Sure , I will do it again.
I am sorry for the unnecessary addition of lines of code I thought was necessary.
Thankyou for your detailed explanation.

Reply:
No worries at all, we all learn together from our mistakes. Keep going!

Reply:
I have edited the files. 
Please let me know is this okay or should I create a new PR for the changes you explained.

Reply:
There is no need to create a new PR, this is good!